clients/lsp-fsharp.el

```diff
@@ -205,6 +205,16 @@ current runtime."
                   "fsautocomplete.netcore.zip"
                 "fsautocomplete.zip")))
 
+(defun lsp-fsharp--change-permissions (install-dir-full)
+  (unless (eq system-type 'windows-nt) ; Windows does not have chmod
+    (progn
+      (message "Altering permissions")
```

Comment:
typically, we use lsp--info instead of message.

Reply:
Updated. Consistency IS important :)